### DISCUSSION TITLE
Some filesystems encrypted: error

### DISCUSSION THREAD

**Discussion by tarkhil** [zrepl/zrepl#706] [Q&A]
Hello

I have a box with one filesystem encrypted; I'm trying to backup it with zrepl.

```
  send:
    encrypted: true
```

set on source, but on destination I get

`rpc error: code = Unknown desc = validate send arguments: encrypted send mandated by policy, but filesystem "bipgames/bootenv/13rc1" is not encrypted`

How do I set up replication with only SOME of filesystems encrypted?

**Comment by problame** [comment]
Not possible. Configure two separate pairs of replication jobs. They need separate root_fs on the receiving side job.

**Reply by tarkhil** [reply]
I see. Why isn't possible to make encrypted optional on fs-bound?
